<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainEscultor</class>
 <widget class="QMainWindow" name="MainEscultor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>562</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>10</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainEscultor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>302</width>
      <height>444</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutR">
           <item>
            <widget class="QLabel" name="labelR">
             <property name="text">
              <string>R:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxR">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderR">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutG">
           <item>
            <widget class="QLabel" name="labelG">
             <property name="text">
              <string>G:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxG">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderG">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutB">
           <item>
            <widget class="QLabel" name="labelB">
             <property name="text">
              <string>B:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderB">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutAlpha">
           <item>
            <widget class="QLabel" name="labelAlpha">
             <property name="text">
              <string>Alpha:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxAlpha">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderAlpha">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="colorRGB" name="widgetrRGB" native="true">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovo"/>
  </widget>
  <widget class="QToolBar" name="toolBarAdd">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Voxel"/>
   <addaction name="actionAdd_Caixa"/>
   <addaction name="actionAdd_Esfera"/>
   <addaction name="actionAdd_Elipsoide"/>
  </widget>
  <widget class="QToolBar" name="toolBarCorta">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCorta_Voxel"/>
   <addaction name="actionCorta_Caixa"/>
   <addaction name="actionCorta_Esfera"/>
   <addaction name="actionCorta_Elipsoide"/>
  </widget>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAdd_Voxel">
   <property name="text">
    <string>Add Voxel</string>
   </property>
  </action>
  <action name="actionCorta_Voxel">
   <property name="text">
    <string>Corta Voxel</string>
   </property>
  </action>
  <action name="actionAdd_Caixa">
   <property name="text">
    <string>Add Caixa</string>
   </property>
  </action>
  <action name="actionCorta_Caixa">
   <property name="text">
    <string>Corta Caixa</string>
   </property>
  </action>
  <action name="actionAdd_Esfera">
   <property name="text">
    <string>Add Esfera</string>
   </property>
  </action>
  <action name="actionCorta_Esfera">
   <property name="text">
    <string>Corta Esfera</string>
   </property>
  </action>
  <action name="actionAdd_Elipsoide">
   <property name="text">
    <string>Add Elipsoide</string>
   </property>
  </action>
  <action name="actionCorta_Elipsoide">
   <property name="text">
    <string>Corta Elipsoide</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>colorRGB</class>
   <extends>QWidget</extends>
   <header>colorrgb.h</header>
   <container>1</container>
   <slots>
    <slot>setR(int)</slot>
    <slot>setG(int)</slot>
    <slot>setB(int)</slot>
    <slot>setAlpha(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderAlpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxAlpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetrRGB</receiver>
   <slot>setR(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetrRGB</receiver>
   <slot>setG(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetrRGB</receiver>
   <slot>setB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetrRGB</receiver>
   <slot>setAlpha(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
